### PR TITLE
test: networks and testnet mode e2e

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)

testnet mode e2e

- enable/disable testnet from settings
- enable/disable testnet shortcut from settings
- enable/disable testnet from menu
- enable/disable testnet from shortcut

networks

- testnet / not testnet
- disabling networks won't show you an option to connect to it

### CODE REVIEW

File: e2e/parallel/networksAndTestnetModeFlow.test.ts [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+      driver,
+      dappHandler,
+    });
+
+    await driver.switchTo().window(popupHandler);
+
+    await clickAcceptRequestButton(driver);
+
+    await driver.switchTo().window(dappHandler);
+    const topButton = await querySelector(
```
Comment (BrodyHughes) [inline e2e/parallel/networksAndTestnetModeFlow.test.ts:113]: could use `findElementByTestId` here

File: e2e/parallel/networksAndTestnetModeFlow.test.ts [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+      driver,
+      '[data-testid="rk-account-button"]',
+    );
+
+    expect(topButton).toBeTruthy();
+    await waitAndClick(topButton, driver);
+
+    const ensLabel = await querySelector(driver, '[id="rk_profile_title"]');
```
Comment (BrodyHughes) [inline e2e/parallel/networksAndTestnetModeFlow.test.ts:121]: could use `findElementById` here

File: e2e/parallel/networksAndTestnetModeFlow.test.ts [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    const dappHandler = await getWindowHandle({ driver });
+
+    const button = await findElementByText(driver, 'Connect Wallet');
+    expect(button).toBeTruthy();
+    await waitAndClick(button, driver);
+
+    const modalTitle = await findElementByText(driver, 'Connect a Wallet');
+    expect(modalTitle).toBeTruthy();
+
+    const mmButton = await querySelector(
```
Comment (BrodyHughes) [inline e2e/parallel/networksAndTestnetModeFlow.test.ts:97]: could use `findElementByTestId` here

**Review by BrodyHughes — APPROVED** [review]
LGTM!

**Review by derHowie — APPROVED** [review]
🔥 🔥

File: e2e/parallel/networksAndTestnetModeFlow.test.ts (hunk 1 above)
Comment (BrodyHughes) [reply e2e/parallel/networksAndTestnetModeFlow.test.ts:113]: yeah, i replaced all three instances of this locally and got `✓ should be able to connect to bx test dapp 16439ms`

Comment (estebanmino) [reply e2e/parallel/networksAndTestnetModeFlow.test.ts:113]: ok i copy pasted this from other tests and it doesn't seem to be working for me

image

will skip for this PR

**Pull request merged**